generators/app/tasks/installDependencies.js

```diff
@@ -112,8 +114,9 @@ module.exports = function installDependencies() {
     DEPENDENCIES.push('react-native-huawei-protected-apps');
   }
 
-  if (this.features.drawerios || this.features.drawerandroid) {
-    DEPENDENCIES.push('react-native-drawer');
+  if (this.features.drawer) {
+    DEPENDENCIES.push('react-navigation-drawer');
+    DEPENDENCIES.push('react-native-reanimated');
```

Comment:
This dependency is repeated, remove it from here or from the `DEPENDENCES` array.